libs/features/profile/src/lib/components/profile-edit/profile-edit.service.ts

```diff
@@ -1,15 +1,19 @@
-import { Injectable, inject } from '@angular/core';
+import { Injectable, computed, inject, signal } from '@angular/core';
 import { Router } from '@angular/router';
+import { FirebaseError } from 'firebase/app';
+import { catchError, of, tap } from 'rxjs';
 
 import { AuthService, ContextService, ErrorService } from '@expenses-tracker/core';
 import { NotificationService, RoutePaths } from '@expenses-tracker/shared/common';
 import { IFlag } from '@expenses-tracker/shared/interfaces';
 
+export type EditMode = 'displayName' | 'password';
+
 export type ComponentFlags = {
-  edit: IFlag;
+  edit: { displayName: IFlag; password: IFlag };

```

Comment:
Use flags in the component

Reply:
Resolved in https://github.com/usamazansari/expenses-tracker/pull/62/commits/1bf47dc47a6e3117892d00c9b2316ef2f40759cc